java8/src/main/java/Client/LogicalOperations.java

```diff
@@ -0,0 +1,184 @@
+package Client;
+
+import Data.Car;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LogicalOperations {
+
+    static List<Car> getObjects(Car t1, Car t2, Car t3, Car t4, Car t5, Car t6) {
+
+        List<Car> list = List.of(t1, t2, t3, t4, t5, t6);
+        return list;
+    }
+
+    static HashSet<Car> filterCarsWithTriggers(List<Car> carslist) {
+        HashSet<Car> targetSet = new HashSet<>();
+        List<Car> intermediaryList = carslist.stream()
+                .filter(car -> car.getModel().startsWith("M"))
+                .collect(Collectors.toList());
+
+        for (Car car : intermediaryList) {

```

Comment:
well you could have done another filter or && condition with 'a', easier, much less code